api/envoy/api/v2/auth/cert.proto

```diff
@@ -171,6 +171,9 @@ message CertificateValidationContext {
   // certificate has not been revoked by this CRL. If this DataSource contains
   // multiple CRLs, all of them will be used.
   core.DataSource crl = 7;
+
+  // If specified, Envoy will not reject expired certs.
+  google.protobuf.BoolValue verify_certificate_expiration = 8;
```

Comment:
Please use bool instead of google.protobuf.BoolValue, and rename so that false is the default, eg "disable_certificate_expiration_verification" or similar.